client/src/actions/ConfigurationsActions.ts

```diff
@@ -117,6 +119,31 @@ class ConfigurationsActions extends AbstractActions implements IConfigurationsAc
     };
   }
 
+  saveAsTemplate(template: IDashboardConfig) {
+    
+    return (dispatcher: (result: { template: IDashboardConfig }) => void) => {
+      let script = this.objectToString(template);
+      
+      script = '/// <reference path="../../../client/@types/types.d.ts"/>\n' +
+              'import * as _ from \'lodash\';\n\n' +
+              'export const config: IDashboardConfig = /*return*/ ' + script;
+      request('/api/templates/' + template.id, {
+          method: 'PUT',
+          json: true,
+          body: { script: script }
+        }, 
+              (error: any, json: any) => {
```

Comment:
You're right, but the linter didn't catch that for some reason.
Tried removing /* tslint:disable-line */ from the file, but not change in result.